src/olmo_core/optim/adamw.py

```diff
@@ -147,6 +147,7 @@ class SkipStepAdamWConfig(OptimConfig):
     betas: Tuple[float, float] = (0.9, 0.999)
     eps: float = 1e-8
     weight_decay: float = 1e-2
+    fused: Optional[bool] = None
```

Comment:
There's no fused option for skip step

Reply:
It takes that as a parameter though.

Reply:
Oh, it shouldn't because it doesn't actually use that param, as well as `foreach`.

Reply:
done